e2e/rescue/claim.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { expect, test } from "@playwright/test";
+import type { Page } from "@playwright/test";
+import fs from "fs";
+
+import dict from "../../src/i18n/i18n";
+import {
+    elementsSendToAddress,
+    generateBitcoinBlock,
+    generateLiquidBlock,
+    getBitcoinAddress,
+    waitForNodesToSync,
+} from "../utils";
+
+const getCurrentSwapId = (page: Page) => {
+    const url = new URL(page.url());
+    return url.pathname.split("/").pop();
+};
+
+const createChainSwap = async (page: Page) => {
+    await page.goto("/");
+
+    const assetSelector = page.locator("div[class='asset asset-LN'] div");
+    await assetSelector.click();
+
+    const lbtcAsset = page.locator("div[data-testid='select-L-BTC']");
+    await lbtcAsset.click();
+
+    const receiveAmount = "0.01";
+    const inputReceiveAmount = page.locator(
+        "input[data-testid='receiveAmount']",
+    );
+    await inputReceiveAmount.fill(receiveAmount);
+
+    const inputSendAmount = page.locator("input[data-testid='sendAmount']");
+    const sendAmount = "0.01003057";
```

Comment:
Don't hardcode values that depend on the backend config. This is not a test for amount calculation